conf/modules.config

```
@@ -31,4 +27,32 @@ process {
         ]
     }
 
+    /* raresidease:
```

Glad I could help!

```suggestion
    /* raredisease:
```

Jokes aside! Is this some type of comment to show what the settings are in the rare-disease pipeline?

---

yes, exactly!